packages/dds/tree/src/test/utils.ts

```diff
@@ -450,6 +452,7 @@ export function spyOnMethod(
 }
 
 /**
+ * TODO
```

Comment:
TODO :)

Reply:
This PR hasn't made changes to `isDeltaVisible()` but the liter is throwing `Missing JSDoc block description ` error. 

I will try to understand what this function does and include the change in this PR.

Reply:
Oh, in that case feel free to disregard. That linter issue is just a warning and can be ignored. It won't actually block the PR. It's just there to remind folks to document their code (until all of the existing violations have been fixed and we can promote the rule to an error).